### ISSUE TITLE
bug in code to obtain minor shower dates

### ISSUE THREAD

**Issue by markmac99** [markmac99/UKmon-shared#305]
Still a problem with this...
```
$  /home/ec2-user/prod/analysis/showerReport.sh AUD 2023
processing AUD for 2023
Traceback (most recent call last):
  File "/home/ec2-user/prod/ukmon_pylib/reports/reportActiveShowers.py", line 215, in <module>
    shwrs = reportActiveShowers(targdate, cml_args.shower, cml_args.thismonth, cml_args.includeminor)
  File "/home/ec2-user/prod/ukmon_pylib/reports/reportActiveShowers.py", line 186, in reportActiveShowers
    shwrname = showerAnalysis(shwr, int(dtstr))
  File "/home/ec2-user/prod/ukmon_pylib/analysis/showerAnalysis.py", line 490, in showerAnalysis
    maxdt = sl.getEnd(shwr) + datetime.timedelta(days=10)
  File "/home/ec2-user/miniconda3/envs/wmpl/lib/python3.8/site-packages/meteortools/fileformats/imoWorkingShowerList.py", line 126, in getEnd
    enddate = datetime.datetime.strptime(shower['peak'], '%b %d') + datetime.timedelta(days=3)
TypeError: strptime() argument 1 must be str, not None
```

**Comment by markmac99** [comment]
Fixed via a hacky workaround but needs to be fixed properly by changes to MeteorTools. 

TLDR is that we should be separately loading the IMO, Full Stream and GMN datasets, and then selecting which set to use, instead of a mishmash of the data.  I'll raise a ticket in MeteorTools to capture this.

**Comment by markmac99** [comment]
closing this as its worked around and needs a fix in the library
